working-with-fhir-igs/hl7v2-to-dk-medcom/README.md

```diff
@@ -0,0 +1,156 @@
+# Ballerina HL7v2 to Danish FHIR Converter
+
+This project demonstrates how to use Ballerina's HL7v2 to FHIR conversion utilities along with the MedCom240 Ballerina library to convert standard HL7v2 messages into Danish FHIR resources (referring MedCom Implementation Guide version 2.4.0.)
+
+## Overview
+
+Healthcare interoperability in Denmark requires conformance to the MedCom FHIR profiles. This utility simplifies the process of converting legacy HL7v2 messages into MedCom-compliant FHIR resources, enabling seamless integration between legacy systems and modern FHIR-based healthcare platforms.
+
+## Features
+
+- Convert HL7v2 ADT messages to MedCom FHIR resources
+- Support for MedCom Implementation Guide v2.4.0 profiles
+- Validation against Danish healthcare requirements
+- Comprehensive error handling and logging
+
+## Prerequisites
+
+- [Ballerina](https://ballerina.io/downloads/) 2201.12.3 (Swan Lake) or newer
+- Basic knowledge of HL7v2 and FHIR standards
+- Knowledge of Danish healthcare standards and MedCom profiles
+
+## HTTP Service
+The transformation runs as an HTTP service. You can invoke the relevant endpoints with a valid HL7v2 ADT_A01 message to see the transformed FHIR resources. This allows for easy integration with existing systems that need to convert legacy HL7v2 messages to FHIR.
+
+## Try Out
+
+1. Clone this repository:
+   ```bash
+   git clone https://github.com/ballerina-guides/healthcare-samples.git
+   cd working-with-fhir-igs/hl7v2-to-dk-medcom
+   ```
+2. Install dependencies:
+   ```bash
+   bal build
+   ```
+3. Start the server
+   ```bash
+   bal run
+   ```
+
+### V2toFHIR Transformation with Danish profiles
+```bash
+curl --location 'http://localhost:9090/hl7/parse/international' \
```

Comment:
Need to have correct paths

Reply:
fixed in [4a378ec](https://github.com/ballerina-guides/healthcare-samples/pull/38/commits/4a378ec4f64c338c69ab412bcd053c7d95901093)